.github/ISSUE_TEMPLATE/security.md

```diff
@@ -0,0 +1,10 @@
+---
+name: 👮 Security Issue
+about: Responsible Disclosure
+---
+
+## 👮 [Responsible Disclosure](https://github.com/nodejs/security-wg/blob/master/processes/responsible_disclosure_template.md)
+
+Do not open issues that might have security implications. It is critical that security related issues
+are reported privately so we have time to address them before they become public knowledge. Please
+email all security related issues to the [Fastify Team](https://github.com/fastify/fastify#team).
```

Comment:
We should a) be recommending to got to hackerone/node security-wg or to email myself or tomas.

Reply:
My DM are open on twitter :)
https://twitter.com/delvedor